SAKPaaS/src/app/components/global-dialog/global-dialog.component.html

```diff
@@ -1,13 +1,28 @@
 <h1 mat-dialog-title>{{ data.titleKey | translate }}</h1>
 <div mat-dialog-content>
   <p [innerHTML]="data.messageKey | translate"></p>
+  <p>
+    <mat-checkbox [formControl]="cookieCheckbox" color="primary">{{
```

Comment:
also mein generelles Problem mit den Änderungen ist, dass du die Cookies&Gps-Checkboxen in den `GlobalDialog` fest implementierst. Der Spirit hinter dem Dialog war ja, dass er eine flexible Variante in der Anwendung darstellt, um überall einfach Dialogboxen anzuzeigen. Das ist jetzt so einfach nicht mehr möglich, da ich **immer** Checkboxes anzeige.

Oder sehe ich das falsch, @luisafuss?

Reply:
Also ich würde vielleicht noch eine Eigenschaft im interface `IDialogMessage ` einbauen, ob du Cookies und GPS-Sachen abfragen willst. Ist vielleicht nicht die schöne Variante, sollte aber passen. Und dann fragst du hier einfach ab, ob du das renderst oder nicht (mittels `*ngIf`).